### PR TITLE
Added item in union..

### PR DESCRIPTION
**Pull request by ThatXliner**

### CODE REVIEW

File: discord-stubs/message.pyi [hunk 1]
```diff
@@ -69,7 +70,7 @@ class Message:
     call: Optional[CallMessage]
     reference: Optional[MessageReference]
     mention_everyone: bool
-    mentions: List[Union[User, Member]]
+    mentions: List[Union[User, Member, Client.user]]
```
Comment (bryanforbes) [inline discord-stubs/message.pyi:73]: This should be `ClientUser` from `.user`

Comment (ThatXliner) [reply discord-stubs/message.pyi:73]: Ok, I'm going to fix that in the next commit

**Comment by ThatXliner** [comment]
Hmm, may I also add an isort github action along in this pull request?

**Comment by bryanforbes** [comment]
I'd rather not. If you follow the steps for [setting up for development](https://github.com/bryanforbes/discord.py-stubs#development), it shouldn't allow you to commit if `isort` fails.

**Comment by ThatXliner** [comment]
I can't seem to fix `ERROR: /home/runner/work/discord.py-stubs/discord.py-stubs/discord-stubs/message.pyi Imports are incorrectly sorted and/or formatted.`

File: discord-stubs/message.pyi [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import datetime
 from os import PathLike
 from typing import Any, BinaryIO, List, Optional, Union
+
```
Comment (bryanforbes) [inline discord-stubs/message.pyi:4]: Take this blank line out

File: discord-stubs/message.pyi [hunk 3]
```diff
@@ -17,9 +18,10 @@ from .mentions import AllowedMentions
 from .partial_emoji import PartialEmoji
 from .reaction import Reaction
 from .role import Role
-from .user import User
+from .user import ClientUser, User
 from .utils import cached_slot_property
 
+
```
Comment (bryanforbes) [inline discord-stubs/message.pyi:24]: Remove this extra blank line

**Comment by bryanforbes** [comment]
I added comments that will fix the `isort` error message in CI

**Comment by bryanforbes** [comment]
Thanks for your contribution! I just released 1.5.0.2 which includes your fix.

**Pull request merged**